web_widget_float_formula/static/tests/js/test_web_widget_float_formula.js

```diff
@@ -13,6 +13,7 @@ odoo.define_section('web_widget_float_formula', ['web.form_common', 'web.form_wi
         self.field = new form_widgets.FieldFloat(field_manager, filler);
         self.$element = $('<input>');
         self.field.$el.append(self.$element);
+        self.field.$label = $('<label>');
```

Comment:
For the last two tests (which tested the change on blur and focus), this function was called: https://github.com/odoo/odoo/blob/6466aa499fc37bba5f4feeeecac99448ef88e62e/addons/web/static/src/js/views/form_common.js#L700 but _toggle_label failed because there was no this.$label in the test, so I put one in.